Ignoring characters preceding start code

thomas-brunel opened the pull request:
Ignore any characters preceding the start code, and remove any space in the line.

I don't know why have two others commit should have done something bad, it was the first time I did a PR. My bad.

thomas-brunel committed:
Ignore every character preceding start code

From the wikipedia page all characters preceding the start code should be ignored. Allowing user to add comment
Add reader test to ensure that characters preceding the start code are correctly ignored

Signed-off-by: Thomas BRUNEL <thmsbru@gmail.com>

thomas-brunel commented:
Fix issue #16

martinmroz commented:
Thanks for doing this, I'll think about whether to release this as a major or minor version. In theory this could break some assumptions clients rely on (i.e. some folks may have built integrations assuming that garbage before the ':' would fail, even though that is incorrect). I'll land it and release a new version shortly.

thomas-brunel committed:
Removed method replace used to ignore all spaces in the record

A record should not have space in it, so we should not ignore them.
Doing otherwise could cause problem

Signed-off-by: Thomas BRUNEL <thmsbru@gmail.com>

thomas-brunel committed:
Merge branch 'master' into master
